doc/shared/sundomeigest/SUNDomEigEst_Arnoldi.rst

```diff
@@ -0,0 +1,182 @@
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2025, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _SUNDomEigEst.ARNOLDI:
+
+The SUNDomEigEst_ARNOLDI Module
+======================================
+
+The SUNDomEigEst_ARNOLDI implementation of the ``SUNDomEigEstimator`` class performs
+the Arnoldi Iteration method :cite:p:`arnoldi51`; this is an iterative dominant
+eigenvalue estimator that is designed to be compatible with any ``N_Vector``
+implementation that supports a minimal subset of operations (:c:func:`N_VClone()`,
+:c:func:`N_VCloneVectorArray()`, :c:func:`N_VDotProd()`, :c:func:`N_VScale()`, 
+:c:func:`N_VDestroy()`, and :c:func:`N_VDestroyVectorArray()`).
+
+Arnoldi iteration is particularly effective for large, sparse matrices where only 
+the dominant eigenvalue is needed.  It constructs an orthonormal basis of the Krylov 
+subspace
+
+.. math::
+
+   \mathcal{K}_m(A, \mathbf{v}) = \text{span}\{\mathbf{v}, A \mathbf{v}, A^2 \mathbf{v}, \dots, A^{m-1} \mathbf{v}\}
+
+using the Gram-Schmidt process.  The matrix :math:`A` is projected onto this subspace
+to form a small upper Hessenberg matrix :math:`H_m`.  The eigenvalues of :math:`H_m`
+approximate some of the eigenvalues of :math:`A`; the dominant eigenvalue of :math:`A` is
+well-approximated by the dominant eigenvalue of :math:`H_m`.
+
+Arnoldi iteration works for matrices with both real and complex eigenvalues.  It supports
+estimations with a user-specified fixed Krylov subspace dimension (at least 3).  While
+the choice of dimension results in a prefixed amount of memory, it strictly
+determines how good an estimation is.  To improve the estimation accuracy, we have found that 
+preprocessing set by :c:func:`SUNDomEigEst_SetNumPreProcess` is particularly useful.  
+This operation is free from any additional memory requirement and is further explained below.
+
+The matrix :math:`A` is not required explicitly; only a routine that provides an 
+approximation of the matrix-vector product, :math:`Av`, is required.
+
+
+.. _SUNDomEigEst.ARNOLDI.Usage:
+
+SUNDomEigEst_ARNOLDI Usage
+-------------------------------
+
+The header file to be included when using this module is ``sundomeigest/sundomeigest_arnoldi.h``.
+The SUNDomEigEst_ARNOLDI module is accessible from all SUNDIALS solvers *without* linking to the
+``libsundials_sundomeigestarnoldi`` module library.
+
+The header file to be included when using this module is ``sundomeigest/sundomeigest_arnoldi.h``.
+The SUNDomEigEst_ARNOLDI module is accessible from all SUNDIALS solvers *without* linking to the
+``libsundials_sundomeigestarnoldi`` module library after enabling SUNDIALS interfaces to the LAPACK library.
```

Comment:
Whether the previous paragraph is still accurate or not, the text is duplicated here, so something needs to be removed.